README.md

```diff
@@ -33,10 +33,11 @@ This project was made with:
 
 - **[header-background-mobile]**
 - **[Headline]**
+- **[About_Me]**
 - **[Contact_Me]**
 
 ### 🚀 Live Demo  <a name="Live Demo"></a>
-[Link to Live demo] Coming soon!
+[https://emarkees.github.io/Portfolio/]
```

Comment:
- [ OPTIONAL ] I think it would be better if you hyperlink the `live demo` text in the way it shows the text only and not the whole link to make your README looks more professional. i.e.
```
[Live demo](https://emarkees.github.io/Portfolio/)
```